netket/sampler/base.py

```diff
@@ -69,50 +69,6 @@ class Sampler(abc.ABC):
     dtype: DType = struct.field(pytree_node=False, default=float)
     """The dtype of the states sampled."""
 
-    def __pre_init__(
-        self, hilbert: AbstractHilbert, n_chains: Optional[int] = None, **kwargs
-    ):
-        """
-        Construct a Monte Carlo sampler.
-
-        Args:
-            hilbert: The Hilbert space to sample.
```

Comment:
do we lose these docstring?

Reply:
the dataclass docstring above contains essentially all of the same information already. Also this one was in my understanding never seen by the users, as it was overridden in the child classes

Reply:
it was shown here https://netket.readthedocs.io/en/latest/api/_generated/samplers/netket.sampler.Sampler.html#netket.sampler.Sampler but now you removed (see https://netket--1511.org.readthedocs.build/en/1511/api/_generated/samplers/netket.sampler.Sampler.html#netket.sampler.Sampler )

Can you just leave the method defined and simply return `(Hilbert, n_chains), kwargs`, in order to have the docstring?

Reply:
I reverted the constructor in the base to the previous behaviour in case anybody outside was using it, wdyt?

Reply:
thanks